src/index.js

```diff
@@ -1,17 +1,38 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import arrayShuffle from "array-shuffle"
+import React from "react"
+import ReactDOM from "react-dom"
+import { BrowserRouter, Route, Switch } from "react-router-dom"
+import ContributorDetails from "./ContributorDetails"
+import entries from "./entries"
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const generateRoutes = (entries) => {
+    if(entries.length === 0){
+        return null
+    } 
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+    return [
+        <Route path="/" key="default" exact>
+            {entries[0].component}
+        </Route>,
+        ...entries.map((entry) => (
+            <Route path={`/${entry.slug}`} key={entry.slug}>
+                {entry.component}
+            </Route>
+        ))
+    ]
```

Comment:
```js
const [initialRoute, ...otherRoutes] = entries

return [
    <Route path="/" key="default" component={initialRoute.component} exact />,
     ...otherRoutes.map((entry) => (
        <Route key={entry.slug} path={`/${entry.slug}`}  component={entry.component} />
      ))
    ]
```

Reply:
I'll do the component part. The logic intended for this is to add the first as the default while still making the slug of the first one available as a route.